cmd/v/help/self.txt

```diff
@@ -0,0 +1,5 @@
+Usage: v self
+
+Rebuild V with the passed options.
+
+Options: All other option are passed to the rebuild command. (e.g. -prod)
```

Comment:
"option" -> "options", "rebuild" -> "build"

There is no "rebuild" command, just "build".